server/commonTransactions/setCellType/setCellType.ts

```diff
@@ -0,0 +1,54 @@
+import FormWizard from 'hmpo-form-wizard'
+import { NextFunction, Response } from 'express'
+import { TypedLocals } from '../../@types/express'
+import FormInitialStep from '../../controllers/base/formInitialStep'
+
+export default class SetCellType extends FormInitialStep {
+  getCellPath = (req: FormWizard.Request, res: Response, index: number) => {
+    const { pathHierarchy } = res.locals.decoratedResidentialSummary.location
+    const newLocationCode = req.sessionModel.get<string>(`locationCode`)
+    const cellNumber = req.sessionModel.get<string>(`create-cells_cellNumber${index}`).padStart(3, '0')
+
+    return [pathHierarchy, newLocationCode, cellNumber].filter(s => s).join('-')
+  }
+
```

Comment:
This can be removed and imported from '../createCells/getCellPath' instead now :)